src/pages/signin/UnlinkLoginForm.js

```diff
@@ -0,0 +1,118 @@
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import redirectToSignIn from '../../libs/actions/SignInRedirect';
+import Avatar from '../../components/Avatar';
+import * as ReportUtils from '../../libs/ReportUtils';
+import networkPropTypes from '../../components/networkPropTypes';
+import {withNetwork} from '../../components/OnyxProvider';
+import DotIndicatorMessage from '../../components/DotIndicatorMessage';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** The credentials of the logged in person */
+    credentials: PropTypes.shape({
+        /** The email/phone the user logged in with */
+        login: PropTypes.string,
+    }),
+
+    /** The details about the account that the user is signing in with */
+    account: PropTypes.shape({
+        /** Whether or not a sign on form is loading (being submitted) */
+        loading: PropTypes.bool,
+
+        /** Whether or not the account is validated */
+        validated: PropTypes.bool,
+
+        /** The primaryLogin associated with the account */
+        primaryLogin: PropTypes.string,
+    }),
+
+    /** Information about the network */
+    network: networkPropTypes.isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    account: {},
+    credentials: {},
+};
+
+const UnlinkLoginForm = (props) => {
+    const primaryLogin = Str.isSMSLogin(props.account.primaryLogin) ? Str.removeSMSDomain(props.account.primaryLogin) : props.account.primaryLogin;
+    const secondaryLogin = Str.isSMSLogin(props.credentials.login) ? Str.removeSMSDomain(props.credentials.login) : props.credentials.login;
+
+    return (
+        <>
+            <View style={[styles.mt3, styles.flexRow, styles.alignItemsCenter, styles.justifyContentStart]}>
+                <Avatar
+                    source={ReportUtils.getDefaultAvatar(props.credentials.login)}
+                    imageStyles={[styles.mr2]}
+                />
+                <View style={[styles.flex1]}>
+                    <Text style={[styles.textStrong]}>
+                        {secondaryLogin}
+                    </Text>
+                </View>
+            </View>
+            <View style={[styles.mt5]}>
+                <Text>
+                    {props.translate('unlinkLoginForm.toValidateLogin', {primaryLogin, secondaryLogin})}
+                </Text>
+            </View>
+            <View style={[styles.mv5]}>
+                <Text>
+                    {props.translate('unlinkLoginForm.noLongerHaveAccess', {primaryLogin})}
+                </Text>
+            </View>
+            {!_.isEmpty(props.account.message) && (
+
+                // DotIndicatorMessage mostly expects onyxData errors so we need to mock an object so that the messages looks similar to prop.account.errors
+                <DotIndicatorMessage style={[styles.mb5, styles.flex0]} type="success" messages={{0: props.account.message}} />
```

Comment:
Should we display either of one - message or error?
Is it possible to have both of them at a same time?

Reply:
I thought i had both here for a reason but looking at it now i think we can get by with only the latter one that checks `props.account.error`

Reply:
Oh wait yes, now I remember. We need this for the `link Sent!` message after the user clicks the `Unlink` button. They shouldn't be able to see both at the same time but I'll test just to be certain

Reply:
Yep, from local testing the user should only be able to see one or the other!

Reply:
Thanks! makes sense